web/tests/Web.E2ETests/Pages/School/BenchmarkWorkforcePage.cs

```diff
@@ -36,13 +36,24 @@ public class BenchmarkWorkforcePage(IPage page)
     private ILocator SaveAsImageButtons =>
         page.Locator(Selectors.Button, new PageLocatorOptions { HasText = "Save as image" });
 
+    private ILocator ComparatorSetDetails =>
+        page.Locator(Selectors.GovDetailsSummaryText, new PageLocatorOptions { HasText = "How we choose similar schools" });
+
+    private ILocator ViewChangeComparatorSetLink => page.Locator(Selectors.GovLink,
```

Comment:
Please rename variable - overlay verbose. "ViewChange" doesn't add any additional value and makes the code harder to read.